src/index.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <title>Activity Tracker</title>
   <!-- <link rel="stylesheet" href="normalize.css"> -->
-  <!-- <link rel="stylesheet" href="styles.scss"> -->
+  <link rel="stylesheet" href="styles.scss">
```

Comment:
Since we are using webpack we do not need to link the stylesheet to the HTML - it just needs to be imported into the script.js file - adding the link to the HTML throws an error in the console.